### ISSUE TITLE
Merged branches should be deleted

### ISSUE THREAD

**Issue by Physicliar** [bounswe/2021SpringGroup8#166]
Merged branches should be updated. Now it is really hard to follow branches and manage them. Pulled and merged branches should be deleted so it will be easy to team members to follow projects.

**Comment by aydinmustafacan** [comment]
At this point project has lots of unused branches. I will delete the branches that are redundant so that it will be easier to follow.

**Comment by MEHMETTEMIZEL** [comment]
> Merged branches should be updated. Now it is really hard to follow branches and manage them. Pulled and merged branches should be deleted so it will be easy to team members to follow projects.

Yeah, that is a very important situation. When I progress slowly in the project, the created and merged branches confuse me in the subject of following the project.

**Comment by aydinmustafacan** [comment]
I have deleted all the redundant branches. Now it is much cleaner. Branches Frontend/Create-community-profile-page , feature/signup-login-android are under development and active. This issue is resolved and can be closed.
